src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-products/sw-sales-channel-detail-products.html.twig

```diff
@@ -92,6 +92,7 @@
                         <slot v-bind="{ item }">
                             {% block sw_sales_channel_detail_products_listing_column_name %}
                             <sw-product-variant-info
+                                :show-tooltip="false"
```

Comment:
tbh, I do not like just disabling it without removing or fixing it. This will be dead code for all eternity and nobody will notice it.

Had a quick look. In line 92 of its `index.js`, we need this

```js
        getFirstSlot() {
            const label = this.$slots?.default?.()?.[0]?.children;

            return typeof label === 'string' ? label : label?.default?.()?.[0]?.children || '';
        },
```

Somehow the nesting got deeper. Do you want to use this? Or should I open a PR instead? 🙂

Reply:
@marcelbrode ah, thank you. 

Yes I agree, that the root issue should be fixed as well, hence I mentioned it :-)
Should this be contained inside this PR, or as a separate one?

Additionally I am not sure, why the tooltip is needed in general, as no additional information is provided in that case? :thinking: